stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/chat/navigation/GalleryImageAttachmentDestination.kt

```diff
@@ -25,10 +25,9 @@ public class GalleryImageAttachmentDestination(
             return
         }
 
-        val userName = message.user.name
         val createdAt = message.getCreatedAtOrThrow().time
         val attachmentIndex = message.attachments.indexOf(attachment)
 
-        start(AttachmentGalleryActivity.createIntent(context, userName, createdAt, attachmentIndex, imageUrls))
+        start(AttachmentGalleryActivity.createIntent(context, createdAt, attachmentIndex, message, attachments))
```

Comment:
@samiuelson "Filtering" is what we do at line 20 I mean. Seems we don't need `imageUrls` anymore, we have to filter attachments instead and check that the filtered collection isn't null. And then pass it to an activity

Reply:
Right, fixed.